locales-pending/premium.ftl

```diff
@@ -247,3 +247,55 @@ false-door-test-content-part-two = Automatically remove data from sites trying t
 false-door-test-content-part-two-dashboard = Auto-delete data from sites trying to sell it.
 false-door-test-cta = Count me in
 false-door-test-popup-close = Close
+
+# High Risk Data Breaches
+
+high-risk-breach-heading = Here’s what to do
+high-risk-breach-subheading = This requires access to your sensitive info, so you’ll need to manually fix it.
+# Variables
+# $num_breaches is the number of breaches where the high risk data was found.
+high-risk-breach-summary = { $num_breaches ->
+  [one] It appeared in { $num_breaches } data breach:
+  *[other] It appeared in { $num_breaches } data breaches:
+}
+# Variables
+# $breach_name is the name of the breach where the high risk data was found.
+# $breach_date is the date when the breach occurred. 
+# An example of this string is Twitter on 13/09/18.
+high-risk-breach-name-and-date = { $breach_name } <breach_date>on { $breach_date }</breach_date>
+high-risk-breach-mark-as-fixed = Mark as fixed
+high-risk-breach-skip = Skip for now
+# Variables:
+# $estimated_time is the estimated time it would take for a user to complete breach resolution steps. It is never singular.
+high-risk-breach-estimated-time = Your estimated time: { $estimated_time }+ minutes
```

Comment:
Please add to the comment that the + will result in "20+" and is  meant as "or more"